packages/contracts/src/EmailAuth.sol

```diff
@@ -88,6 +130,7 @@ contract EmailAuth is OwnableUpgradeable, UUPSUpgradeable {
     }
 
     /// @notice Retrieves a subject template by its ID.
+    /// @dev This function can only be called by the module contract.
```

Comment:
Missing `onlyModule` modifier if this should only be called by the module contract?